Add Jira migration FAQ for decision-makers (#24572) (#24581)
opf/openproject: docs/installation-and-operations/jira-migration/faq/README.md

> 
## Is OpenProject proven in large organizations?

Yes. Many large organizations have been using OpenProject successfully for years. References and major deployments include Deutsche Bahn, Mercedes-AMG, Samsung, 3M, Charité, the German Federal Ministry for Digital Transformation and Government Modernisation (BMDS), and the [International Criminal Court (ICC)](https://www.openproject.org/blog/digital-sovereignty-government-germany-opendesk/) through openDesk.

OpenProject is trusted by organizations across government and public administration, healthcare, transportation, manufacturing, automotive, research and higher education, energy, IT, and consulting. It is increasingly adopted by public sector organizations pursuing digital sovereignty, open standards, and vendor independence. The  [OpenProject customer overview](https://www.openproject.org/customers/) provides further examples across industries and geographies.

> OpenProject is also currently undergoing an extensive security assessment as part of openDesk for use by the German Armed Forces (Bundeswehr). This work goes beyond a standard functional evaluation and examines the software and its operation against the requirements of a particularly security-sensitive organization.

Author: ```suggestion
OpenProject is also currently undergoing an extensive security assessment as part of openDesk for use in highly security-sensitive organizations. This work goes beyond a standard functional evaluation and examines the software and its operation against the requirements of such environments.
```
